src/Management/test/Endpoint.Test/ActuatorRouteBuilderExtensionsTest.cs

```diff
@@ -153,10 +153,12 @@ private async Task ActAndAssertAsync(IHostBuilder builder, Type endpointOptionsT
         response.StatusCode.Should().Be(expectSuccess ? HttpStatusCode.OK : HttpStatusCode.Forbidden);
     }
 
+#pragma warning disable SA1602 // Enumeration items should be documented
```

Comment:
Why was this added? It wasn't needed before, right?

Reply:
I added it because I was getting SA1602 _errors_ here. I thought that was odd, but didn't think it was worth trying to understand or document

Reply:
Where did you get the errors exactly? I see them in VS, but running a build in release mode (both from VS and `dotnet build`) doesn't produce errors.

![image](https://github.com/user-attachments/assets/6379428b-b2cd-4d1a-b777-091c2b0d350f)

Either way, it's pretty unusual to have public enums in test projects. However, in this case, it's required for `TheoryData`. A better way to suppress is to add `SA1602` to `NoWarn` in `shared-test.props`. Because we'd never want errors in test projects for that.